packages/ckeditor5-table/src/tablecolumnresize/utils.js

```diff
@@ -357,11 +357,21 @@ export function insertColumnResizerElements( viewWriter, viewCell ) {
 // @param {module:engine/view/element~Element} viewCell View cell.
 export function removeColumnResizerElements( viewWriter, viewCell ) {
 	const viewTableColumnResizerElement = [ ...viewCell.getChildren() ]
-		.find( viewElement => viewElement.hasClass( 'table-column-resizer' ) );
+		.find( viewElement => viewElement.hasClass( 'ck-table-column-resizer' ) );
 
 	if ( !viewTableColumnResizerElement ) {
 		return;
 	}
 
 	viewWriter.remove( viewTableColumnResizerElement );
 }
+
+//
```

Comment:
Looks like someone wanted to document something here 😉

Reply:
I would consider naming this method like something that includes "outter width" or something. Just to emphasize that it includes nonstandard stuff like padding or border.

BTW, what about margin?

Reply:
> I would consider naming this method like something that includes "outter width" or something

:+1: 

> BTW, what about margin?

According to the [CSS spec](https://www.w3.org/TR/CSS2/tables.html#table-layout) (and [live experiment](https://codepen.io/dumluregn/pen/MWQMbwE) :P), margin is ignored for the table cell elements.